examples/tutorials/04_cubes_and_objects/12_roll_cube.py

```diff
@@ -18,11 +18,16 @@
 
 This example demonstrates Cozmo driving to and rolling a cube.
 You must place a cube in front of Cozmo so that he can see it.
+The cube should be no farther than 5 inches away from Cozmo
```

Comment:
Why is there a 5 inch limit? Just because that's where it breaks, or is it intentional that it doesn't work past that far away.